src/pages/ShopDetail/components/ImageCarousel/index.tsx

```diff
@@ -0,0 +1,52 @@
+import useCounter from 'utils/hooks/useCounter';
+import { ReactComponent as PrevIcon } from 'assets/svg/shop/prev-arrow.svg';
+import { ReactComponent as NextIcon } from 'assets/svg/shop/next-arrow.svg';
+import styles from './ImageCarousel.module.scss';
+
+interface Props {
+  imageUrls: string[];
+}
+
+function ImageCarousel({ imageUrls }: Props) {
+  const { count: currentIndex, increment, decrement } = useCounter(0);
+
+  const next: React.ReactEventHandler<HTMLButtonElement> = () => {
```

Comment:
이벤트를 안쓴다면 굳이 정의하지 않아도 될 것 같아요
```suggestion
  const handleNext(Click?) = () => {
```

Reply:
오호! 습관처럼 쓰고 있었는데, 지적 감사합니다~!